dashboard/dashboard-front/src/components/jobs/deployment/DeploymentDetails.vue

```diff
@@ -0,0 +1,126 @@
+<script setup lang="ts">
+import { ref, type Ref, onMounted } from 'vue'
+import { useRoute } from 'vue-router'
+import { toastService } from '@/services/ToastService'
+import { apiClient } from '@/services/ApiClient'
+import {formatDuration, timestampToLocalTime} from '@/utils/time'
+import {type DeploymentDto} from '@/utils/api-schema'
+import TimeAgoLabel from "@/components/jobs/TimeAgoLabel.vue"
+import ManifestView from "@/components/jobs/ManifestView.vue"
+
+const route = useRoute()
+const deploymentId = route.params.deploymentId
+const deployment: Ref<DeploymentDto | null> = ref(null)
```

Comment:
It doesn't make much difference but you could also use generics of `ref` function instead of importing `Ref` type. `const deployment = ref<DeploymentDto | null>(null)`